base/ltcmd.dtx

```diff
@@ -3996,19 +4008,15 @@
           }
       }
     \use:n
-      {
-        \tl_trim_spaces_apply:nN {#1} \@@_arg_to_keyvalue_auxi:nnn
-          {#2} {#1}
-      }
+      { \@@_arg_to_keyvalue_auxi:nnn {#1} {#2} {#3} }
```

Comment:
Why the `\use:n { \@@_arg_to_keyvalue_auxi:nnn {#1} {#2} {#3} }`? Just use `\@@_arg_to_keyvalue_auxi:nnn` directly and use `\use_none:nnnn` instead of `\use_none:nn`.